tools/run-mypy

```diff
@@ -68,8 +74,11 @@ if not python_files and not pyi_files:
     sys.exit(0)
 
 mypy_args: List[str] = []
-if args.quiet:
+# --no-error-summary is not supported by the daemon
+if args.quiet and not args.use_daemon:
     mypy_args += ["--no-error-summary"]
+if args.use_daemon:
+    mypy_args += ["--status-file", STATUS_FILE, "run"]
```

Comment:
The custom status file makes it annoying to interact with `dmypy` manually, e.g. to stop the daemon.